colabs/Zmiana_typu_danych.ipynb

```diff
@@ -0,0 +1,985 @@
+            "Collecting pyspark\n",
+            "  Downloading pyspark-3.3.1.tar.gz (281.4 MB)\n",
+            "\u001b[K     |████████████████████████████████| 281.4 MB 38 kB/s \n",
+            "\u001b[?25hCollecting py4j\n",
+            "  Downloading py4j-0.10.9.7-py2.py3-none-any.whl (200 kB)\n",
+            "\u001b[K     |████████████████████████████████| 200 kB 58.6 MB/s \n",
+            "\u001b[?25h  Downloading py4j-0.10.9.5-py2.py3-none-any.whl (199 kB)\n",
+            "\u001b[K     |████████████████████████████████| 199 kB 56.5 MB/s \n",
+            "\u001b[?25hBuilding wheels for collected packages: pyspark\n",
+            "  Building wheel for pyspark (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Created wheel for pyspark: filename=pyspark-3.3.1-py2.py3-none-any.whl size=281845512 sha256=168b7698f2ef81777a7bcccb7964c535c5fc9441f47331f5ab1aae9d86395b1c\n",
+            "  Stored in directory: /root/.cache/pip/wheels/42/59/f5/79a5bf931714dcd201b26025347785f087370a10a3329a899c\n",
+            "Successfully built pyspark\n",
+            "Installing collected packages: py4j, pyspark\n",
+            "Successfully installed py4j-0.10.9.5 pyspark-3.3.1\n",
+            "--2022-11-24 01:08:01--  ftp://ftp.ps.pl/pub/apache/spark/spark-3.3.1/spark-3.3.1-bin-hadoop2.tgz\n",
+            "           => ‘spark-3.3.1-bin-hadoop2.tgz’\n",
+            "Resolving ftp.ps.pl (ftp.ps.pl)... 212.14.28.36, 2001:4c58:62:2::36\n",
+            "Connecting to ftp.ps.pl (ftp.ps.pl)|212.14.28.36|:21... connected.\n",
+            "Logging in as anonymous ... Logged in!\n",
+            "==> SYST ... done.    ==> PWD ... done.\n",
+            "==> TYPE I ... done.  ==> CWD (1) /pub/apache/spark/spark-3.3.1 ... done.\n",
+            "==> SIZE spark-3.3.1-bin-hadoop2.tgz ... 274099817\n",
+            "==> PASV ... done.    ==> RETR spark-3.3.1-bin-hadoop2.tgz ... done.\n",
+            "Length: 274099817 (261M) (unauthoritative)\n",
+            "\n",
+            "spark-3.3.1-bin-had 100%[===================>] 261.40M  21.3MB/s    in 13s     \n",
+            "\n",
+            "2022-11-24 01:08:16 (19.7 MB/s) - ‘spark-3.3.1-bin-hadoop2.tgz’ saved [274099817]\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import pyspark\n",
+        "import findspark\n",
+        "from pyspark.sql import SparkSession\n",
+        "import os\n",
+        "import pyspark.sql.functions as F\n",
+        "import pyspark.sql.types as T\n",
+        "from pyspark.sql.functions import split, col\n",
+        "from pyspark.sql.types import StructType, StringType, IntegerType, BooleanType, FloatType, TimestampType, DateType, ArrayType, MapType\n",
+        "from typing import List, Tuple, Dict, Any\n",
+        "from pyspark.sql.types import StructType, StringType\n",
+        "from pyspark.sql.functions import monotonically_increasing_id \n",
+        "import numpy"
```

Comment:
No to zostaje jeszcze jedno - uporządkowanie tych importów. Zauważmy, że np.
```
from pyspark.sql.types import StructType, StringType, IntegerType, BooleanType, FloatType, TimestampType, DateType, ArrayType, MapType
...
from pyspark.sql.types import StructType, StringType
```
się dublują. Gdzie indziej mamy
```
from pyspark.sql.functions import split, col
from pyspark.sql.functions import monotonically_increasing_id 
``` 
tymczasem wystarczy
```
from pyspark.sql.functions import split, col, monotonically_increasing_id 
```